src/Sfx/App/Scripts/Controllers/SettingsController.ts

```diff
@@ -22,7 +22,7 @@ module Sfx {
 
             this.data.getNodes().then(nodeCollection => {
                 nodeCollection.setAdvancedMode(this.advancedModeState);
-            })
+            });
         }

```

Comment:
Were you able to get the actions to be removed when you toggled them without adding anything here? With the current way we are doing advanced actions, you would want to add a method like the one on node collection for doing the same with the other places that can have backups.